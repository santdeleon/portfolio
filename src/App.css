.rainbow-top {
  background: linear-gradient(
    to right,
    #f76f6f,
    #fcca67,
    #fcfc83,
    #84ffaf,
    #83e0f7,
    #ca67e5,
    #ff87e1
  );
  padding: 5px;
}

.particles-background {
  z-index: -1;
  position: absolute;
}
