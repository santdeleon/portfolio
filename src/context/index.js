import { BrandProvider, BrandContext } from './BrandContext';
import { ThemeProvider, ThemeContext } from './ThemeContext';

export { BrandProvider, BrandContext, ThemeProvider, ThemeContext };
