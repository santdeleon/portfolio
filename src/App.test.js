import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import { ThemeProvider } from './context/ThemeContext';
import { BrandProvider } from './context/BrandContext';

import App from './App';

// TODO: Get the portfolio test coverage up to 100%
describe('the App component', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <BrowserRouter>
        <BrandProvider>
          <ThemeProvider>
            <App />
          </ThemeProvider>
        </BrandProvider>
      </BrowserRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
