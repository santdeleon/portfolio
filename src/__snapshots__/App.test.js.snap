// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the App component should match snapshot 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="App__Div--rainbow-top"
    />
    <nav
      class="NavMenu navbar navbar-expand navbar-light"
    >
      <a
        aria-label="Home"
        class="navbar-brand"
        href="/"
        style="color: rgb(255, 255, 255);"
        title="Home"
      >
        <span
          aria-label="Rainbow Emoji"
          role="img"
        >
          🌈
        </span>
         Sant Deleon
      </a>
      <div
        class="mr-auto d-none d-md-flex navbar-nav"
      >
        <li
          class="mx-2"
        >
          <a
            class="NavLink__Button text-decoration-none p-0 btn btn-link"
            href="#Portfolio"
            style="color: rgb(255, 255, 255);"
          >
            <span
              class="mr-1"
            >
              <svg
                fill="none"
                height="16"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M22 19a2 2 0 0 1-2 2H4a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h5l2 3h9a2 2 0 0 1 2 2z"
                />
              </svg>
            </span>
            Portfolio
          </a>
        </li>
        <li
          class="mx-2"
        >
          <a
            class="NavLink__Button text-decoration-none p-0 btn btn-link"
            href="#Footer"
            style="color: rgb(255, 255, 255);"
          >
            <span
              class="mr-1"
            >
              <svg
                fill="none"
                height="16"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <polyline
                  points="16 18 22 12 16 6"
                />
                <polyline
                  points="8 6 2 12 8 18"
                />
              </svg>
            </span>
            Technologies
          </a>
        </li>
        <li
          class="mx-2"
        >
          <a
            class="NavLink__Button text-decoration-none p-0 btn btn-link"
            href="mailto:sant@santdeleon.io"
            style="color: rgb(255, 255, 255);"
          >
            <span
              class="mr-1"
            >
              <svg
                fill="none"
                height="16"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="20"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M21 11.5a8.38 8.38 0 0 1-.9 3.8 8.5 8.5 0 0 1-7.6 4.7 8.38 8.38 0 0 1-3.8-.9L3 21l1.9-5.7a8.38 8.38 0 0 1-.9-3.8 8.5 8.5 0 0 1 4.7-7.6 8.38 8.38 0 0 1 3.8-.9h.5a8.48 8.48 0 0 1 8 8v.5z"
                />
              </svg>
            </span>
            Get in touch
          </a>
        </li>
      </div>
      <div
        class="ml-auto align-items-center navbar-nav"
      >
        <a
          aria-label="Github"
          class="p-0 mr-2 btn btn-link"
          href="https://github.com/santdeleon"
          rel="noopener noreferrer"
          target="_blank"
          title="Github"
        >
          <img
            alt="Github"
            src="github-dark.svg"
          />
        </a>
        <span
          aria-checked="false"
          aria-label="Activate Light Mode"
          class="ToggleSwitch ToggleSwitch--dark"
          role="checkbox"
          tabindex="0"
          title="Activate Light Mode"
        >
          <div
            class="ToggleSwitch__Div--track"
          />
          <div
            class="ToggleSwitch__Div--thumb ToggleSwitch__Div--thumb-dark"
            style="transform: translateX(18px);"
          >
            <img
              alt="Moon"
              class="w-100"
              src="moon.svg"
            />
          </div>
        </span>
      </div>
    </nav>
    <header
      class="Header mt-5 flex-column container"
      id="Header"
    >
      <div
        class="row"
      >
        <div
          class="col"
        >
          <img
            alt="Sant & Moon"
            class="Header__Img--avatar"
            src="sant-dark.svg"
          />
          <h1
            class="display-4"
          >
            <span
              class="App--rainbow-text"
            >
              Hello,
               
            </span>
            I'm Sant. 
            <br />
            <span
              class="d-none d-md-block"
            >
              Front-end & Blockchain Developer
            </span>
          </h1>
          <h4
            class="d-none d-md-block text-muted font-weight-light mt-3 mb-5"
          >
            with a knack for creating cutting edge products with elegance and style.
             
            <span
              aria-label="Sparkle Emoji"
              role="img"
            >
              ✨
            </span>
          </h4>
          <h4
            class="d-block d-md-none text-muted font-weight-light mb-4"
          >
            front-end developer and blockchain engineer
             
          </h4>
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class="bg-transparent"
              >
                <div
                  class="Header__Div--skill-header"
                  style="background-color: rgb(247, 111, 110);"
                />
                <div
                  class="Header__Div--skill-body text-center py-3"
                >
                  <p
                    class="p-0 m-0 d-none d-lg-block"
                  >
                    Front-end Developer
                  </p>
                  <span
                    class="d-block d-lg-none"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-code fa-w-20 "
                      data-icon="code"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      viewBox="0 0 640 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M278.9 511.5l-61-17.7c-6.4-1.8-10-8.5-8.2-14.9L346.2 8.7c1.8-6.4 8.5-10 14.9-8.2l61 17.7c6.4 1.8 10 8.5 8.2 14.9L293.8 503.3c-1.9 6.4-8.5 10.1-14.9 8.2zm-114-112.2l43.5-46.4c4.6-4.9 4.3-12.7-.8-17.2L117 256l90.6-79.7c5.1-4.5 5.5-12.3.8-17.2l-43.5-46.4c-4.5-4.8-12.1-5.1-17-.5L3.8 247.2c-5.1 4.7-5.1 12.8 0 17.5l144.1 135.1c4.9 4.6 12.5 4.4 17-.5zm327.2.6l144.1-135.1c5.1-4.7 5.1-12.8 0-17.5L492.1 112.1c-4.8-4.5-12.4-4.3-17 .5L431.6 159c-4.6 4.9-4.3 12.7.8 17.2L523 256l-90.6 79.7c-5.1 4.5-5.5 12.3-.8 17.2l43.5 46.4c4.5 4.9 12.1 5.1 17 .6z"
                        fill="currentColor"
                      />
                    </svg>
                  </span>
                </div>
              </div>
            </div>
            <div
              class="col"
            >
              <div
                class="bg-transparent"
              >
                <div
                  class="Header__Div--skill-header"
                  style="background-color: rgb(251, 192, 122);"
                />
                <div
                  class="Header__Div--skill-body text-center py-3"
                >
                  <p
                    class="p-0 m-0 d-none d-lg-block"
                  >
                    Blockchain Engineer
                  </p>
                  <span
                    class="d-block d-lg-none"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-ethereum fa-w-10 "
                      data-icon="ethereum"
                      data-prefix="fab"
                      focusable="false"
                      role="img"
                      viewBox="0 0 320 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M311.9 260.8L160 353.6 8 260.8 160 0l151.9 260.8zM160 383.4L8 290.6 160 512l152-221.4-152 92.8z"
                        fill="currentColor"
                      />
                    </svg>
                  </span>
                </div>
              </div>
            </div>
            <div
              class="col"
            >
              <div
                class="bg-transparent"
              >
                <div
                  class="Header__Div--skill-header"
                  style="background-color: rgb(252, 224, 115);"
                />
                <div
                  class="Header__Div--skill-body text-center py-3"
                >
                  <p
                    class="p-0 m-0 d-none d-lg-block"
                  >
                    UI/UX Designer-ish
                  </p>
                  <span
                    class="d-block d-lg-none"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-palette fa-w-16 "
                      data-icon="palette"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      viewBox="0 0 512 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M204.3 5C104.9 24.4 24.8 104.3 5.2 203.4c-37 187 131.7 326.4 258.8 306.7 41.2-6.4 61.4-54.6 42.5-91.7-23.1-45.4 9.9-98.4 60.9-98.4h79.7c35.8 0 64.8-29.6 64.9-65.3C511.5 97.1 368.1-26.9 204.3 5zM96 320c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32zm32-128c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32zm128-64c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32zm128 64c-17.7 0-32-14.3-32-32s14.3-32 32-32 32 14.3 32 32-14.3 32-32 32z"
                        fill="currentColor"
                      />
                    </svg>
                  </span>
                </div>
              </div>
            </div>
            <div
              class="col"
            >
              <div
                class="bg-transparent"
              >
                <div
                  class="Header__Div--skill-header"
                  style="background-color: rgb(165, 242, 159);"
                />
                <div
                  class="Header__Div--skill-body text-center py-3"
                >
                  <p
                    class="p-0 m-0 d-none d-lg-block"
                  >
                    Tech Enthusiast
                  </p>
                  <span
                    class="d-block d-lg-none"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-robot fa-w-20 "
                      data-icon="robot"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      viewBox="0 0 640 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M32,224H64V416H32A31.96166,31.96166,0,0,1,0,384V256A31.96166,31.96166,0,0,1,32,224Zm512-48V448a64.06328,64.06328,0,0,1-64,64H160a64.06328,64.06328,0,0,1-64-64V176a79.974,79.974,0,0,1,80-80H288V32a32,32,0,0,1,64,0V96H464A79.974,79.974,0,0,1,544,176ZM264,256a40,40,0,1,0-40,40A39.997,39.997,0,0,0,264,256Zm-8,128H192v32h64Zm96,0H288v32h64ZM456,256a40,40,0,1,0-40,40A39.997,39.997,0,0,0,456,256Zm-8,128H384v32h64ZM640,256V384a31.96166,31.96166,0,0,1-32,32H576V224h32A31.96166,31.96166,0,0,1,640,256Z"
                        fill="currentColor"
                      />
                    </svg>
                  </span>
                </div>
              </div>
            </div>
            <div
              class="col"
            >
              <div
                class="bg-transparent"
              >
                <div
                  class="Header__Div--skill-header"
                  style="background-color: rgb(130, 225, 243);"
                />
                <div
                  class="Header__Div--skill-body text-center py-3"
                >
                  <p
                    class="p-0 m-0 d-none d-lg-block"
                  >
                    Coffee Drinker
                  </p>
                  <span
                    class="d-block d-lg-none"
                  >
                    <svg
                      aria-hidden="true"
                      class="svg-inline--fa fa-mug-hot fa-w-16 "
                      data-icon="mug-hot"
                      data-prefix="fas"
                      focusable="false"
                      role="img"
                      viewBox="0 0 512 512"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M127.1 146.5c1.3 7.7 8 13.5 16 13.5h16.5c9.8 0 17.6-8.5 16.3-18-3.8-28.2-16.4-54.2-36.6-74.7-14.4-14.7-23.6-33.3-26.4-53.5C111.8 5.9 105 0 96.8 0H80.4C70.6 0 63 8.5 64.1 18c3.9 31.9 18 61.3 40.6 84.4 12 12.2 19.7 27.5 22.4 44.1zm112 0c1.3 7.7 8 13.5 16 13.5h16.5c9.8 0 17.6-8.5 16.3-18-3.8-28.2-16.4-54.2-36.6-74.7-14.4-14.7-23.6-33.3-26.4-53.5C223.8 5.9 217 0 208.8 0h-16.4c-9.8 0-17.5 8.5-16.3 18 3.9 31.9 18 61.3 40.6 84.4 12 12.2 19.7 27.5 22.4 44.1zM400 192H32c-17.7 0-32 14.3-32 32v192c0 53 43 96 96 96h192c53 0 96-43 96-96h16c61.8 0 112-50.2 112-112s-50.2-112-112-112zm0 160h-16v-96h16c26.5 0 48 21.5 48 48s-21.5 48-48 48z"
                        fill="currentColor"
                      />
                    </svg>
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </header>
    <main
      class="Main"
    >
      <div
        class="Portfolio py-5 my-5 container"
        id="Portfolio"
      >
        <div
          class="py-5 my-5 row"
        >
          <div
            class="ProjectCard mb-lg-0 px-2 mb-5 col-lg-4 col-md-6 col-12"
          >
            <a
              class="text-decoration-none text-light"
              href="https://www.hostgator.com/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                class="ProjectCard__Div--card-wrapper p-3 shadow border border-dark"
              >
                <h3
                  class="mb-0 text-truncate"
                >
                  HostGator
                </h3>
                <div
                  class="row no-gutters"
                >
                  <div
                    class="col-10"
                  >
                    <p
                      class="ProjectCard__P text-truncate"
                    >
                      Domain & Web Hosting
                    </p>
                  </div>
                  <div
                    class="col-2"
                  >
                    <p
                      class="text-right"
                    >
                      <span
                        alt="Crocodile Emoji"
                        aria-label="Crocodile Emoji"
                        role="img"
                      >
                        🐊
                      </span>
                    </p>
                  </div>
                </div>
                <div
                  class="d-flex justify-content-center rounded py-4"
                  style="background: rgb(253, 223, 164);"
                >
                  <img
                    alt="HostGator"
                    src="hostgator.png"
                    style="width: 10rem;"
                  />
                </div>
                <div
                  class="mt-4 row"
                >
                  <div
                    class="col"
                  >
                    <p
                      class="ProjectCard__P mb-0"
                    >
                      Role
                    </p>
                    <p
                      class="ProjectCard__P ProjectCard__P--gradient-text text-truncate"
                    >
                      Software Engineer
                    </p>
                  </div>
                </div>
              </div>
            </a>
          </div>
          <div
            class="ProjectCard mb-lg-0 px-2 mb-5 col-lg-4 col-md-6 col-12"
          >
            <a
              class="text-decoration-none text-light"
              href="https://github.com/santdeleon/coinflip"
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                class="ProjectCard__Div--card-wrapper p-3 shadow border border-dark"
              >
                <h3
                  class="mb-0 text-truncate"
                >
                  Coinflip
                </h3>
                <div
                  class="row no-gutters"
                >
                  <div
                    class="col-10"
                  >
                    <p
                      class="ProjectCard__P text-truncate"
                    >
                      Blockchain Gaming
                    </p>
                  </div>
                  <div
                    class="col-2"
                  >
                    <p
                      class="text-right"
                    >
                      <span
                        alt="Money Bag Emoji"
                        aria-label="Money Bag Emoji"
                        role="img"
                      >
                        💰
                      </span>
                    </p>
                  </div>
                </div>
                <div
                  class="d-flex justify-content-center rounded py-4"
                  style="background: rgb(255, 232, 160);"
                >
                  <img
                    alt="Ethereum"
                    src="ethereum.svg"
                    style="width: 7.5rem;"
                  />
                </div>
                <div
                  class="mt-4 row"
                >
                  <div
                    class="col"
                  >
                    <p
                      class="ProjectCard__P mb-0"
                    >
                      Role
                    </p>
                    <p
                      class="ProjectCard__P ProjectCard__P--gradient-text text-truncate"
                    >
                      UI/UX & Blockchain Engineer
                    </p>
                  </div>
                </div>
              </div>
            </a>
          </div>
          <div
            class="ProjectCard mb-lg-0 px-2 col-lg-4 col-md-6 col-12"
          >
            <a
              class="text-decoration-none text-light"
              href="https://www.terraats.com/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <div
                class="ProjectCard__Div--card-wrapper p-3 shadow border border-dark"
              >
                <h3
                  class="mb-0 text-truncate"
                >
                  Terra ATS
                </h3>
                <div
                  class="row no-gutters"
                >
                  <div
                    class="col-10"
                  >
                    <p
                      class="ProjectCard__P text-truncate"
                    >
                      IT & SaaS Solutions
                    </p>
                  </div>
                  <div
                    class="col-2"
                  >
                    <p
                      class="text-right"
                    >
                      <span
                        alt="Globe Showing Americas Emoji"
                        aria-label="Globe Showing Americas Emoji"
                        role="img"
                      >
                        🌎
                      </span>
                    </p>
                  </div>
                </div>
                <div
                  class="d-flex justify-content-center rounded py-4"
                  style="background: rgb(16, 36, 45);"
                >
                  <img
                    alt="TerraAts"
                    src="terra.png"
                    style="width: 7.5rem;"
                  />
                </div>
                <div
                  class="mt-4 row"
                >
                  <div
                    class="col"
                  >
                    <p
                      class="ProjectCard__P mb-0"
                    >
                      Role
                    </p>
                    <p
                      class="ProjectCard__P ProjectCard__P--gradient-text text-truncate"
                    >
                      Software Engineering Intern
                    </p>
                  </div>
                </div>
              </div>
            </a>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="Footer border-top px-3 px-sm-5 border-dark"
      id="Footer"
    >
      <p
        class="mb-0 py-4"
      >
        <small>
          Website last updated: 
          December 25, 2020
           
          <span
            aria-label="Christmas Tree"
            role="img"
          >
            🎄
          </span>
          <span
            aria-label="Wrapped Gift"
            class="mx-1"
            role="img"
          >
            🎁
          </span>
          <span
            aria-label="Santa Claus"
            role="img"
          >
            🎅
          </span>
        </small>
      </p>
      <h5>
        Languages
      </h5>
      <div
        class="row"
      >
        <div
          class="col-lg-8 col-12"
        >
          <div
            class="row"
          >
            <div
              class="col-sm-3 col-12"
            >
              <ul
                class="p-0"
              >
                <li
                  class="Footer__Li font-weight-bold text-light"
                >
                  Frontend
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  HTML/CSS
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  JavaScript
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  React
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Redux
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  GatsbyJS
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  NodeJS
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  jQuery
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  KnockoutJS
                </li>
              </ul>
            </div>
            <div
              class="col-sm-3 col-12"
            >
              <ul
                class="p-0"
              >
                <li
                  class="Footer__Li font-weight-bold text-light"
                >
                  <span>
                     
                  </span>
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Jest
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Mocha
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Chai
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  SCSS
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Template Toolkit
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Bootstrap
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Storybook
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Lerna
                </li>
              </ul>
            </div>
            <div
              class="col-sm-3 col-12"
            >
              <ul
                class="p-0"
              >
                <li
                  class="Footer__Li font-weight-bold text-light"
                >
                  Blockchain
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Solidity
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Web3
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  @web3/react
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  EthersJS
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  @ethersproject
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Oraclize
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  OpenZepplin
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Ganache
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Truffle Suite
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Ethereum Testnets
                </li>
              </ul>
            </div>
            <div
              class="col-sm-3 col-12"
            >
              <ul
                class="p-0"
              >
                <li
                  class="Footer__Li font-weight-bold text-light"
                >
                  Backend
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Java
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Perl
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  VIM
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  MongoDB
                </li>
                <li
                  class="Footer__Li mt-1"
                >
                  Express
                </li>
              </ul>
            </div>
          </div>
        </div>
        <div
          class="text-left text-lg-right col-lg-4 col-12"
        >
          <p
            class="mb-0 font-weight-light"
          >
            Give me a shout
             
            <a
              aria-label="Contact Sant"
              class="Footer__A--contact font-weight-bold text-decoration-none App--rainbow-text"
              href="mailto:sant@santdeleon.io"
              title="Contact Sant"
            >
              sant@santdeleon.io
            </a>
          </p>
          <p
            class="font-weight-light"
          >
            Let's get
             
            <span
              class="font-weight-bold text-light"
            >
              coffee
            </span>
             
            <span
              aria-label="Coffee Emoji"
              role="img"
            >
              ☕
            </span>
          </p>
        </div>
      </div>
      <nav
        class="p-0 navbar navbar-expand navbar-light"
      >
        <div
          class="mr-auto navbar-nav"
        >
          <p
            class="font-weight-light mb-0 App--rainbow-text"
          >
            Made with
            <span
              aria-label="Two Hearts Emoji"
              class="mx-2"
              role="img"
            >
              💕
            </span>
            by Sant
          </p>
        </div>
        <div
          class="ml-auto navbar-nav"
        >
          <a
            aria-label="GitHub"
            class="text-light nav-link"
            data-rb-event-key="https://github.com/santdeleon"
            href="https://github.com/santdeleon"
            rel="noopener noreferrer"
            target="_blank"
            title="GitHub"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-github fa-w-16 "
              data-icon="github"
              data-prefix="fab"
              focusable="false"
              role="img"
              viewBox="0 0 496 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                fill="currentColor"
              />
            </svg>
          </a>
          <a
            aria-label="LinkedIn"
            class="text-light nav-link"
            data-rb-event-key="https://www.linkedin.com/in/sant-deleon/"
            href="https://www.linkedin.com/in/sant-deleon/"
            rel="noopener noreferrer"
            target="_blank"
            title="LinkedIn"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-linkedin-in fa-w-14 "
              data-icon="linkedin-in"
              data-prefix="fab"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
                fill="currentColor"
              />
            </svg>
          </a>
          <a
            aria-label="Angellist"
            class="text-light nav-link"
            data-rb-event-key="https://angel.co/santdeleon"
            href="https://angel.co/santdeleon"
            rel="noopener noreferrer"
            target="_blank"
            title="Angellist"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-angellist fa-w-14 "
              data-icon="angellist"
              data-prefix="fab"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M347.1 215.4c11.7-32.6 45.4-126.9 45.4-157.1 0-26.6-15.7-48.9-43.7-48.9-44.6 0-84.6 131.7-97.1 163.1C242 144 196.6 0 156.6 0c-31.1 0-45.7 22.9-45.7 51.7 0 35.3 34.2 126.8 46.6 162-6.3-2.3-13.1-4.3-20-4.3-23.4 0-48.3 29.1-48.3 52.6 0 8.9 4.9 21.4 8 29.7-36.9 10-51.1 34.6-51.1 71.7C46 435.6 114.4 512 210.6 512c118 0 191.4-88.6 191.4-202.9 0-43.1-6.9-82-54.9-93.7zM311.7 108c4-12.3 21.1-64.3 37.1-64.3 8.6 0 10.9 8.9 10.9 16 0 19.1-38.6 124.6-47.1 148l-34-6 33.1-93.7zM142.3 48.3c0-11.9 14.5-45.7 46.3 47.1l34.6 100.3c-15.6-1.3-27.7-3-35.4 1.4-10.9-28.8-45.5-119.7-45.5-148.8zM140 244c29.3 0 67.1 94.6 67.1 107.4 0 5.1-4.9 11.4-10.6 11.4-20.9 0-76.9-76.9-76.9-97.7.1-7.7 12.7-21.1 20.4-21.1zm184.3 186.3c-29.1 32-66.3 48.6-109.7 48.6-59.4 0-106.3-32.6-128.9-88.3-17.1-43.4 3.8-68.3 20.6-68.3 11.4 0 54.3 60.3 54.3 73.1 0 4.9-7.7 8.3-11.7 8.3-16.1 0-22.4-15.5-51.1-51.4-29.7 29.7 20.5 86.9 58.3 86.9 26.1 0 43.1-24.2 38-42 3.7 0 8.3.3 11.7-.6 1.1 27.1 9.1 59.4 41.7 61.7 0-.9 2-7.1 2-7.4 0-17.4-10.6-32.6-10.6-50.3 0-28.3 21.7-55.7 43.7-71.7 8-6 17.7-9.7 27.1-13.1 9.7-3.7 20-8 27.4-15.4-1.1-11.2-5.7-21.1-16.9-21.1-27.7 0-120.6 4-120.6-39.7 0-6.7.1-13.1 17.4-13.1 32.3 0 114.3 8 138.3 29.1 18.1 16.1 24.3 113.2-31 174.7zm-98.6-126c9.7 3.1 19.7 4 29.7 6-7.4 5.4-14 12-20.3 19.1-2.8-8.5-6.2-16.8-9.4-25.1z"
                fill="currentColor"
              />
            </svg>
          </a>
          <a
            aria-label="Contact"
            class="text-light nav-link"
            data-rb-event-key="mailto:sant@santdeleon.io"
            href="mailto:sant@santdeleon.io"
            rel="noopener noreferrer"
            target="_blank"
            title="Contact"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-envelope fa-w-16 "
              data-icon="envelope"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
                fill="currentColor"
              />
            </svg>
          </a>
        </div>
      </nav>
    </footer>
  </div>
</div>
`;
